interop/stress/client/main.go

```diff
@@ -276,7 +289,15 @@ func logParameterInfo(addresses []string, tests []testCaseWithWeight) {
 
 func newConn(address string, useTLS, testCA bool, tlsServerName string) (*grpc.ClientConn, error) {
 	var opts []grpc.DialOption
-	if useTLS {
+	if *customCredentialsType != "" {
+		if *customCredentialsType == googleDefaultCredsName {
+			opts = append(opts, grpc.WithCredentialsBundle(google.NewDefaultCredentials()))
+		} else if *customCredentialsType == computeEngineCredsName {
+			opts = append(opts, grpc.WithCredentialsBundle(google.NewComputeEngineCredentials()))
+		} else {
+			logger.Fatalf("Unknown custom credentials: %v", *customCredentialsType)
+		}
```

Comment:
Consider:

```go
switch *customCredentialsType {
	case googleDefaultCredsName:
	case computeEngineCredsName:
	case "":
		if useTLS {
		}
	default:
		// unknown custom creds error
}
```

Also if we can change the spec here, consider `custom_creds_type=="tls"` means to use tls instead of having a separate flag for it.  Having two flags for one setting is problematic.

Reply:
That's how the flags work in the interop test client across languages as well as the java stress test one. Makes sense to revisit, but I think that would be a broad change outside the scope of this PR.